tools/ui-components/src/color-system.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
Great trick.
I wonder if we should be pulling these colors directly from tailwind once we have the variables setup?

Reply:
The color names are fine for now but could introduce a naming convention down the road for the colors.css. Since the number next to the color is referred to the mix ratio of the colors in the gray white gradient instance, we might need to separate the colors that are pure from the colors that are mixed to be used for  background and foreground.

Additionally a double digit for the alpha of the translucent colors.

Reply:
> I wonder if we should be pulling these colors directly from tailwind once we have the variables setup?

Are you referring to the tokens in the Tailwind config? Yeah I think it would make more sense to display those tokens instead on Storybook, as we kind of want to hide the `colors.css` layer and try to communicate using semantic names.

Though I guess we can show both ¯\\_(ツ)\_/¯ . I would think it's convenient for you still, to be able to check how a color would fit when developing the new palette. I initially didn't plan to display these on Storybook yet, but had to eventually, as I needed to give the variables in `colors.css` a new name and I couldn't tell which should be which without the ramp 😛